docs/components_page/components/input/colorpicker.py

```diff
@@ -0,0 +1,22 @@
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import dash_bootstrap_components as dbc
+
+colorpicker = dbc.FormGroup(
+    [
+        dbc.Label("Choose the text color:", id="label"),
+        dbc.Input(
+            type="color",
+            id="colorpicker",
+            value="#000000",
+            style={"width": 75, "height": 50},
+        ),
+    ]
+)
+
+
+@app.callback(
+    Output("label", "style"), Input("colorpicker", "value"),
```

Comment:
Can you remove the trailing comma after `Input`? If you run `black` on this it will explode the arguments onto multiple lines but it fits quite neatly onto 1 line here

Reply:
done